Exiled.API/Extensions/ItemExtensions.cs

```diff
@@ -177,5 +178,81 @@ public static List<ItemType> GetItemTypes(this IEnumerable<Item> items)
             itemTypes.AddRange(items.Select(item => item.Type));
             return itemTypes;
         }
+
+        /// <summary>
+        /// Gets all <see cref="AttachmentIdentifier"/>s present on an <see cref="ItemType"/>.
+        /// </summary>
+        /// <param name="type">The <see cref="ItemType"/> to iterate over.</param>
+        /// <param name="code">The <see cref="uint"/> value which represents the attachments code to check.</param>
+        /// <returns>A <see cref="IEnumerable{T}"/> of <see cref="AttachmentIdentifier"/> value which represents all the attachments present on the specified <see cref="ItemType"/>.</returns>
+        public static IEnumerable<AttachmentIdentifier> GetAttachments(this ItemType type, uint code)
+        {
+            code -= (uint)type.GetBaseCode();
```

Comment:
This is not intended to be used on public plugins, if you're gonna use this shit, it's probably because you know what you're doing. However I'm gonna add this check.